tests/logger/test_log_collector.py

```diff
@@ -224,342 +225,348 @@ def _enter_leave_block_section_all_df(self):
+            log_collector.get_block_section_times_of_train(event_bus.run_id, "ice_2"),
             _enter_leave_block_section_2_df,
         )
 
     def test_enter_leave_block_section_3(
-        self, _enter_leave_block_section_3_df, logger, log_collector: LogCollector
+        self, _enter_leave_block_section_3_df, event_bus, log_collector: LogCollector
     ):
-        self.setup_enter_leave_block_section_3(logger)
+        self.setup_enter_leave_block_section_3(event_bus)
         assert_frame_equal(
-            log_collector.get_block_section_times_of_train(logger.run_id, "ice_3"),
+            log_collector.get_block_section_times_of_train(event_bus.run_id, "ice_3"),
             _enter_leave_block_section_3_df,
         )
 
     def test_enter_leave_block_section_4(
-        self, _enter_leave_block_section_4_df, logger, log_collector: LogCollector
+        self, _enter_leave_block_section_4_df, event_bus, log_collector: LogCollector
     ):
-        self.setup_enter_leave_block_section_4(logger)
+        self.setup_enter_leave_block_section_4(event_bus)
         assert_frame_equal(
-            log_collector.get_block_section_times_of_train(logger.run_id, "ice_4"),
+            log_collector.get_block_section_times_of_train(event_bus.run_id, "ice_4"),
             _enter_leave_block_section_4_df,
         )
 
     def test_departure_arrival_all(
-        self, _departure_arrival_all_df, logger, log_collector: LogCollector
+        self, _departure_arrival_all_df, event_bus, log_collector: LogCollector
     ):
-        self.setup_departure_arrival_1(logger)
-        self.setup_departure_arrival_2(logger)
-        self.setup_departure_arrival_3(logger)
-        self.setup_departure_arrival_4(logger)
+        self.setup_departure_arrival_1(event_bus)
+        self.setup_departure_arrival_2(event_bus)
+        self.setup_departure_arrival_3(event_bus)
+        self.setup_departure_arrival_4(event_bus)
         assert_frame_equal(
-            log_collector.get_departures_arrivals_all_trains(logger.run_id),
+            log_collector.get_departures_arrivals_all_trains(event_bus.run_id),
             _departure_arrival_all_df,
         )
 
     def test_departure_arrival_all_1(
-        self, _departure_arrival_all_1_df, logger, log_collector: LogCollector
+        self, _departure_arrival_all_1_df, event_bus, log_collector: LogCollector
     ):
-        self.setup_departure_arrival_1(logger)
+        self.setup_departure_arrival_1(event_bus)
         assert_frame_equal(
-            log_collector.get_departures_arrivals_all_trains(logger.run_id),
+            log_collector.get_departures_arrivals_all_trains(event_bus.run_id),
             _departure_arrival_all_1_df,
         )
 
-    def test_departure_arrival_empty(self, logger: Logger, log_collector: LogCollector):
+    def test_departure_arrival_empty(
+        self, event_bus: Logger, log_collector: LogCollector
```

Comment:
Why is it `event_bus: Logger`? Does this make the typehints better or is there a real reason?

Reply:
We just overlooked this. It will be changed :)